README.md

```diff
@@ -1,43 +1,55 @@
-# piqueserver [![Build Status](https://travis-ci.org/piqueserver/piqueserver.svg?branch=master)](https://travis-ci.org/piqueserver/piqueserver)
+# :spades: :cloud: piqueserver [![Build Status](https://travis-ci.org/piqueserver/piqueserver.svg?branch=master)](https://travis-ci.org/piqueserver/piqueserver)
 An Ace of Spades 0.75 server based on [PySnip](https://github.com/NateShoffner/PySnip).
 
-## FAQ
-> **What's the purpose?**
->
-> The purpose of this repo is to be a continuation of PySnip.
+## :rocket: Installation
+```bash
+virtualenv -p python2 venv && . venv/bin/activate && pip install -r requirements.txt
+./setup.py install
+```
+And lift off! 
+```
+piqueserver -d configs
+```
+## :speech_balloon: FAQ
 
-<br>
+#### What's the purpose?
 
-> **What if PySnip development returns?**
->
-> Then they would merge our changes and development would be continued
-> there, I guess. The important thing is to keep AoS servers alive.
 
-<br>
+ The purpose of this repo is to be a continuation of PySnip.
 
-> **How long will it take for the first release?**
->
-> March 2016 probably. 
 
-<br>
+#### What if PySnip development returns?
 
-> **Why should I use piqueserver instead of PySnip/PySpades?**
-> 
-> * bug fixes
-> * improvements
-> * better anti-hacking
-> * new scripts
+Then they would merge our changes and development would be continued
+there, I guess. The important thing is to keep AoS servers alive.
 
-<br>
 
-> **What about 0.76 support**
->
-> Working with multiple versions is a pain. 0.76 will be suported in the
-> future only.
+#### How long will it take for the first release?
 
-<br>
+March 2016 probably. 

```

Comment:
@VelocityRa Oops!

Looks like we didn't got used to 2017 yet :P.

Going to fix it right now!